references/cli/addon-registry.go

```diff
@@ -277,6 +279,8 @@ func parseArgsFromFlag(cmd *cobra.Command) {
 	cmd.Flags().StringP(addonOssBucket, "", "", "specify the OSS bucket name")
 	cmd.Flags().StringP(addonPath, "", "", "specify the addon registry OSS path")
 	cmd.Flags().StringP(addonGitToken, "", "", "specify the github repo token")
+	cmd.Flags().StringP(addonUsername, "", "", "specify the addon registry username")
+	cmd.Flags().StringP(addonPassword, "", "", "specify the addon registry password")
```

Comment:
Please point out  clearly the username and password args only can be used for helm repo registry.